### DISCUSSION TITLE
Ability to Add Story Points

### DISCUSSION THREAD

**Discussion by dragoste17** [community/community#5241] [Projects and Issues]
Whenever I create an issue, I feel the need to have some measure for the difficulty of the issue.
Using story points the difficulty, can be communicated better to the rest of the team. Also, we can use it for reviewing the performance of our employees depending on how many story points they have completed.

**Comment by ericboucher** [comment]
Agreed @dragoste17! As a workaround for now we built https://github.com/oviohub/github-story-points which you can then use as a plugin for chrome and firefox to enhance the GitHub project board.

- [**Chrome** extension][link-cws] [<img valign="middle" src="https://img.shields.io/chrome-web-store/v/macalhhnmnhceejcjhbgdfidpeeamhep.svg?label=%20">][link-cws]
- [**Firefox** add-on][link-amo] [<img valign="middle" src="https://img.shields.io/amo/v/github-project-story-points.svg?label=%20">][link-amo]

It is an evolution of https://github.com/banyan/github-story-points

[link-cws]: https://chrome.google.com/webstore/detail/github-story-points/macalhhnmnhceejcjhbgdfidpeeamhep 'Version published on Chrome Web Store'
[link-amo]: https://addons.mozilla.org/en-US/firefox/addon/github-project-story-points/ 'Version published on Mozilla Add-ons'

**Comment by zwass** [comment]
An important consideration is that story points should be attached to the issue, not just the project. Currently custom fields are lost when issues are moved between projects.

**Comment by densto51** [comment]
Start
